### DISCUSSION TITLE
paddleocr训练数据支持使用文本块吗,就是那种日本漫画里的一整个气泡框,因为使用了manga109数据集

### DISCUSSION THREAD

**Discussion by zeroling** [PaddlePaddle/PaddleOCR#13708] [Q&A]
因为manga109数据集的text标注是对整个文本块进行标注,并不是一行或者一列的标注方式,我可以直接用文本块的数据训练吗

**Comment by GreatV** [comment]
PaddleOCR在默认情况下并不直接支持对整个文本块（如日本漫画中的对话气泡）进行处理。PaddleOCR的标准流程通常是对较小的、分段的文本区域进行检测和识别，而不是对整个文本块进行统一的处理。

要使用PaddleOCR处理这种大块的文本块，你可能需要做以下几件事：

1. **自定义模型或训练流程**：可以尝试自定义训练流程，或者调整模型架构，使其能够更好地处理和识别整个文本块。

2. **预处理文本块**：在训练前，可以考虑先对大块文本进行分割，将整个文本块切割成多个行或列，转换为PaddleOCR更容易处理的格式。

目前，最直接的方法可能还是对文本块进行一些预处理，然后再使用PaddleOCR进行识别。不过，如果想实现对整个文本块的直接识别，可能需要更为复杂的模型定制和开发工作。
